### PR TITLE
Feature/java11 neo4jv4

### PR DESCRIPTION
**Pull request by DirkMahler**

### CODE REVIEW

File: plugins/pom.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@
             <groupId>com.buschmais.jqassistant.plugin</groupId>
             <artifactId>java</artifactId>
         </dependency>
+        <!-- todo java 11 -->
```
Comment (StephanPirnbaum) [inline plugins/pom.xml:53]: ?

File: status.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Java 11
```
Comment (StephanPirnbaum) [inline status.txt:1]: Please remove this file

File: test/src/test/java/com/buschmais/jqassistant/commandline/test/AbstractCLIIT.java [hunk 3]
```diff
@@ -30,12 +30,12 @@
  */
 public abstract class AbstractCLIIT {
 
-    public enum NeoVersion {
-        NEO4JV3("neo4jv3");
+    public enum Neo4jVersion {
```
Comment (StephanPirnbaum) [inline test/src/test/java/com/buschmais/jqassistant/commandline/test/AbstractCLIIT.java:33]: Do we actually need this enum if there is only one option anyway?

**Review by StephanPirnbaum — CHANGES_REQUESTED** [review]

File: test/src/test/java/com/buschmais/jqassistant/commandline/test/AbstractCLIIT.java (hunk 3 above)
Comment (DirkMahler) [reply test/src/test/java/com/buschmais/jqassistant/commandline/test/AbstractCLIIT.java]: Removed, furthermore removed neo4jv4 suffix from CLI distribution

File: plugins/pom.xml (hunk 1 above)
Comment (StephanPirnbaum) [reply plugins/pom.xml:53]: still present

**Review by StephanPirnbaum — CHANGES_REQUESTED** [review]

File: plugins/pom.xml (hunk 1 above)
Comment (StephanPirnbaum) [reply plugins/pom.xml]: Nevermind, was just me not using Github properly...

**Review by StephanPirnbaum — APPROVED** [review]

**Review by obfischer — APPROVED** [review]

**Pull request merged**